CONTRIBUTING.rst

```diff
@@ -200,6 +200,69 @@ can become the Mentor and guide the proposed candidates on how they can become a
     the mentor must try to remain impartial and cannot be the Proposer.
 
 
+Inactive Committers
+-------------------
+If you know you are not going to be able to contribute for a long time
+(for instance, due to a change of job or circumstances), you should inform the PMC and we will mark you
+as "inactive". Inactive committers will be removed from the "roster" on ASF and will no longer have the power
+of being a Committer (especially write access to the repos). As merit earned never expires, once you
+become active again you can simply email the PMC and ask to be reinstated.
+
+The PMC also can mark committers as inactive after they have not been involved in the community for
+more than 12 months.
+
+
+Guidelines to become an Airflow Committer
+------------------------------------------
+
+Committers are community members who have write access to the project’s
+repositories, i.e., they can modify the code, documentation, and website by themselves and also
+accept other contributions. There is no strict protocol for becoming a committer. Candidates for new
+committers are typically people that are active contributors and community members.
+
+Some people might be active in several of those areas and while they might have not enough 'achievements' in any
+single one of those, their combined contributions in several areas all count.
+
+As a community, we appreciate contributions to the Airflow codebase, but we also place equal value
+on those who help Airflow by improving the community in some way. It is entirely possible to become
+a committer (and eventually a PMC member) without ever having to change a single line of code.
+
+
+Prerequisites
+^^^^^^^^^^^^^^
+
+General prerequisites that we look for in all candidates:
+
+1.  Consistent contribution over last few months
+2.  Visibility on discussions on the dev mailing list and/or the Slack channel
```

Comment:
Just for complement sake, should we add something like:
```suggestion
2.  Visibility on discussions on the dev mailing list, Slack channels or Github issues/discussions
```

Btw `and/or == or` 😄

Reply:
:D indeed

Reply:
Updated in e729fc1